CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - Added support for resolving superclass properties for not-NSObject subclasses
 - The `{% for %}` tag can now iterate over tuples, structures and classes via
   their stored properties.
+- You can now use brackets in boolean expressions to change operators precedence
```

Comment:
Could you update the entry format to match the rest? (`.`, 2 spaces, credit yourself and ref. this PR)